echidna/core/dsextract.py

```diff
@@ -452,11 +685,13 @@ class Radial3ExtractMC(Extractor):
         :class:`echidna.calc.constants` is used in the calculation.
 
     Attributes:
-      _outer_radius (float, optional): The fixed radius used in calculating
+      _outer_radius (float): The fixed radius used in calculating
         :math:`(radius/outer_radius)^3`.
```

Comment:
There are a couple of places where you've put `outer_radius` in `:math:`. This shows up a little weirdly in the docs - see image.
![screenshot from 2015-10-16 12 32 50](https://cloud.githubusercontent.com/assets/1931666/10540696/4be3a3e8-7402-11e5-96fb-07a50c5fa3fa.png)
I think :math:`(radius/outer\_radius)^3` should solve this.